pom.xml

```diff
@@ -1253,6 +1253,7 @@
     <commonsLoggingVersion>99.99.99-use-jcl-over-slf4j</commonsLoggingVersion>
     <commonsMath3Version>3.5</commonsMath3Version>
     <commonsValidatorVersion>1.3.1</commonsValidatorVersion>
+    <curatorVersion>2.0.0-incubating</curatorVersion>
```

Comment:
This version seems old, can you upgrade to a later version? 2.10.0, 3.0.0, or 3.1.0 are available.